### DISCUSSION TITLE
Restrict length rule on PostalZone elements

### DISCUSSION THREAD

**Discussion by rkottmann** [OP-TED/eForms-SDK#866] [Ideas]
Currently, PostZone element content length is restricted to 400 characters after whitespace normalization:

e.g. 
```xml
<rule context="/*/cac:BusinessParty/cac:PostalAddress/cbc:PostalZone">
  <assert id="BT-512-Business_length" role="ERROR" 
             test="string-length(normalize-space(.)) le 400">rule|text|BT-512-Business_length</assert>
</rule>
```
However, the Document on "General Adressing Issues" from the Universal Postal Union states that the maximum length is currently 10 digits (see  https://www.upu.int/UPU/media/upu/documents/PostCode/General-Addressing-Issues.pdf question 2).

Therefore, I would like to propose to significantly lower the length to much less than 20 or even 10.

**Comment by YvesJo** [comment]
Hi,
Indeed, at publication time of the UPU document, the longest postcodes were composed of 10 characters and it could be relevant to have stricter rules about the code length.
I'm also converting the issue in a discussion of type ideas, which appears as more appropriate.
